README.md

```diff
@@ -15,6 +15,39 @@ for Windows' [Google Group](http://groups.google.com/group/git-for-windows),
 and [contribute bug
 fixes](https://github.com/git-for-windows/git/wiki/How-to-participate).
 
+To build Git for Windows, please either install [Git for Windows'
+SDK](https://gitforwindows.org/#download-sdk), start its `git-bash.exe`, `cd`
+to your Git worktree and run `make`, or open the Git worktree as a folder in
+Visual Studio.
+
+To verify that your build works, use one of the following methods:
+
+- If you want to test the built executables within Git for Windows' SDK,
+  prepend `<worktree>/bin-wrappers` to the `PATH`.
```

Comment:
Do they need to remember to export the new value of PATH?

Maybe add something to explain what this does or how it works.
```
This directory contains a set of wrapper scripts that mirror the real Git commands. They customize low-level Git environment variables and then launch the real, locally built Git executables directly from the worktree root directory.
```

Reply:
> Do they need to remember to export the new value of PATH?

No. Once a variable has been exported, it stays exported. There are no two values, one for the local and one for the exported variable. It is just one variable ;-)

> Maybe add something to explain what this does or how it works.
> 
> ```
> This directory contains a set of wrapper scripts that mirror the 
> real Git commands. They customize low-level Git environment
> variables and then launch the real, locally built Git executables
> directly from the worktree root directory.
> ```

This strikes me as too much of an implementation detail. It is unnecessary to know about this when you just want to test the built executables.